frontend/src/components/TutorPage.js

```diff
@@ -81,15 +82,11 @@ export default function TutorPage() {
             },
           }}
         >
-          <Avatar
-            sx={{
-              mt: { md: "5px" },
-              bgcolor: blue[500],
-              width: { xs: 50, md: 100 },
-              height: { xs: 50, md: 100 },
-            }}
-            alt={data.tutor.firstname + " " + data.tutor.lastname}
-          />
+          <div className="gravatar">
+            <img
+              src={'https://www.gravatar.com/avatar/' + md5((data.tutor.username).toLowerCase()) + '?f=y&d=robohash'}
```

Comment:
Usernames are case-sensitive, we shouldn't be converting them to lowercase.

Reply:
Will fix. Nice catch